components/booking/picker_form.blade.php

```diff
@@ -18,6 +18,7 @@
                 data-days-of-week-disabled='@json($disabledDaysOfWeek ?? [])'
                 data-dates-disabled='@json($disabledDates ?? [])'
                 data-format="yyyy-mm-dd"
+                data-language={{ setting('default_language') }}
```

Comment:
should it be : 
data-language= "{{ strtolower(str_replace('_', '-', setting('default_language'))) }}"
instead ?

Reply:
Handle it in the JS - updated:
https://github.com/tastyigniter/TastyIgniter/pull/724